utils/tweet_query.py

```diff
@@ -95,6 +98,10 @@ def pull(self, query:str, output_dir: str,
 				print('\n' + '-'*30)
 				break
 
+			# Avoiding 1 request/sec rate limit
+			end_time_req = time.time()
+			time.sleep(max(0, 1.0 - (end_time_req-start_time_req)))
```

Comment:
Fixed negative sleep value issue, and moved start_time_req line (81).  Before, it was executed before the call to Tweepy, so the rate limit was still exceeded in rare instances where the time before the request was actually made was significant.